### ISSUE TITLE
windows 11

### ISSUE THREAD

**Issue by anilkadian081** [nwoolls/MultiMiner#464]
After trying to install and run in windows after disabling windows defender it still wasn't running and having issue identifing mining hardware

**Comment by Firelight97** [comment]
I don't think you need to disable your Defender. Just tell it to allow the program to run on the device.
